docs/package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "vuepress dev",
     "build": "vuepress build"
   },
-  "license": "MIT",
+  "license": "SEE LICENSE IN LICENSE",
```

Comment:
Do we need to mention the EE license for the docs?

Reply:
Will have to see with sales for this ;) not urgent I think. WIll setup CLA too and improve the contributing guide I think

and FYI, it was for a legal stuff that I got asked to remove the `md`. No idea why atm

Reply:
🤔  interesting, noted.